src/test/java/pages/HomePage.java

```diff
@@ -2,7 +2,6 @@
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import sun.jvm.hotspot.asm.Register;
 
 public class HomePage extends BasePage{
 
```

Comment:
better make baseUrl variable as a private static final

Reply:
@vasilly
I just refactor most of the suggestions you recommended. Please check again. I will try to remember all your comments for homework 6.